blurring_as_a_service/performace_evaluation_pipeline/components/evaluate_with_cvt_metrics.py

```diff
@@ -1,14 +1,16 @@
 import os
+import pathlib
 import sys
 
-from mldesigner import Input, command_component
-
-from blurring_as_a_service.metrics.custom_metrics_calculator import (
-    collect_and_store_tba_results_per_class_and_size,
-)
+from mldesigner import Input, command_component  # noqa: E402
 
 sys.path.append("../../..")
 
+
+from blurring_as_a_service.metrics.custom_metrics_calculator import (  # noqa: E402
+    CustomMetricsCalculator,
+    collect_and_store_tba_results_per_class_and_size,
```

Comment:
Because it is another metrics
the custom metrics calculator is actually computing false negative rates mainly and the total blur area is doing something else, they also have different inputs. 

Im thinking now to rename the custom metrics calculator to fnr_metric_calculator
and put together everything related to total blurred area into tba_metric_calculator.

what do you think?